src/common/components/notification/toastNotification.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { toast, ToastOptions } from 'react-toastify';
+
+import styles from './toastNotification.module.css';
+import NotificationComponent from './NotificationComponent';
+
+const options: ToastOptions = {
+  autoClose: false,
+  className: styles.toastBody,
+  bodyClassName: styles.toastBody,
+};
+
+export type NotificationOptions = {
+  labelText?: string;
+  notificationMessage?: string;
+  type?: 'info' | 'success' | 'alert' | 'error';
+  onClose?: () => void;
+};
+
+const toastNotification = (notificationOptions: NotificationOptions) => {
```

Comment:
This is stored under `components`. Is it really a component?

Reply:
True! Moved it under helpers instead.